doc/manual/src/release-notes/index.md

```diff
@@ -1,12 +1,15 @@
 # Nix Release Notes
 
-Nix has a release cycle of roughly 6 weeks.
-Notable changes and additions are announced in the release notes for each version.
+The Nix release cycle is calendar-based as follows:
+
+- A new minor version (`XX.YY+1.0`) is published every month and supported for two months;
+- A new major version (`XX+1.1.0`) is published twice a year, in April and October, and supported for eight months.
```

Comment:
I don't understand what this means. What are `XX` and `YY`? Year and month? And what do the `+1.0` and `+1.1.0` denote?